packages/ButtonLink/ButtonLink.jsx

```diff
@@ -43,6 +43,20 @@ const StyledButtonLink = styled(StyledButton)(
         color: hoverColor,
       },
     }
+  },
+  ({ fullWidth }) => {
+    let width
+    if (fullWidth === 'true') {
```

Comment:
Can you write this as a ternary statement instead?

Reply:
Hi @mike-bunce,
I have updated PR again with requested changes